lms/templates/view.html

```diff
@@ -28,6 +28,13 @@ <h1>תצוגת תרגיל {{ solution['exercise']['id'] }}: {{ solution['exercis
       {% endfor -%}
       </ul>
     </nav>
+    {% if is_manager %}
+        <div class="progress" style="width: 100%;direction: ltr">
```

Comment:
Don't include inline styles (add them in the CSS instead)

Reply:
Use \<progress\>

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress